lib/chorde/shmemutils.py

```diff
@@ -352,15 +352,26 @@ def close(self):
         if bitmap is not None:
             # Release slot
             self.bitmap_slot.value = False
+        del bitmap
```

Comment:
Is this necessary?

Reply:
Yes, we can't still have references to the bitmap when we .close() it